os/os-rtems.h

```diff
@@ -0,0 +1,71 @@
+/*-
+ * Copyright (c) 2018 Udit kumar agarwal <dev.madaari@gmail.com>
+ * Copyright (c) 2018 Christian Mauderer <christian.mauderer@embedded-brains.de>
```

Comment:
It's nice that you mention me but I'm not aware that I did anything in that file. So I think you can just drop that line.